### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by fchamicapereira**
Fetching the sessions from deck and put them in a schedule.

To test this, just change the event variable in the fetch.js from '25-sinfo' to '24-sinfo'.

### CODE REVIEW

**Comment by johnytiago** [comment]
The result looks great! GJ
One thing I think we should add is min-height to the presentation's description to make it easier to read and compare sessions.
screen shot 2018-01-31 at 20 12 40

What do you think?

File: sinfo25/assets/js/fetch.js [hunk 1]
```diff
@@ -1,22 +1,133 @@
...
+  var deck = 'https://deck.sinfo.org:443/api/';
   var request = new XMLHttpRequest();
   request.open('GET', deck + field + '?' + params);
   request.responseType = 'json';
   request.send();
   request.onload = function() {
     request.response.forEach(function(el){
-      processDataFromDeck(el);
+      if (extraData) {
+        processDataFromDeck(el, extraData);
```
Comment (johnytiago) [inline sinfo25/assets/js/fetch.js]: To prevent "ugly" `if ... else`, you can do 
```
if (something) 
  return foo
return baz
```

**Comment by fchamicapereira** [comment]
Like this?
screen shot 2018-02-01 at 03 13 27

**Comment by carlosfaria94** [comment]
Nice job @fchamicapereira ! :)

**Pull request merged**